web/libs/ui/src/tailwind.config.js

```diff
@@ -190,6 +209,9 @@ module.exports = {
         lg: "var(--radius)",
         md: "calc(var(--radius) - 2px)",
         sm: "calc(var(--radius) - 4px)",
+
+        // Add all border radii from tokens
```

Comment:
```suggestion
        // Add all border radius from tokens
```

Reply:
radii is the plural form here, otherwise it reads weird using the singular `radius` term.